Migrations/20190715184242_InitialMigration.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
```

Comment:
Is it normal to add Migrations into github?

Reply:
Looks like yes.